### ISSUE TITLE
Potential regression with skip-link rule

### ISSUE THREAD

**Issue by thepassle** [dequelabs/axe-core#2069]
Hi!

At open-wc we provide a project generator, that scaffolds a basic starter application using web components. In this setup, we also scaffold automated accessibility tests that relies on axe-core.

We recently ran into an issue where the accessibility tests started failing where they didnt before, from the changelog, we're not quite sure at which version exactly this was introduced, but our best guess is this code here: https://github.com/dequelabs/axe-core/commit/6a1bcba#diff-ee6303bd768770ef023bc5e2ace0c650R23-R25

The HTML it's failing on is:
```html
<header>
  <ul>
    <li>
      <a href="#main" class=${this.__navClass('main')} @click=${this.__onNavClicked}>
        Main
      </a>
    </li>
    <li>
      <a href="#pageOne" class=${this.__navClass('pageOne')} @click=${this.__onNavClicked}>
        Page One
      </a>
    </li>
    <li>
      <a href="#about" class=${this.__navClass('about')} @click=${this.__onNavClicked}>
        About
      </a>
    </li>
  </ul>
</header>
```

Which you can find [here](https://github.com/open-wc/open-wc/blob/master/packages/create/src/generators/app-lit-element/templates/_MyApp.js#L86), and is a very naive implementation of hash based routing that renders a component once a link is clicked. (The project setup is intentionally simple so users can easily make it their own)

Axe seems to have started identifying these as skiplinks though, and fails with: 

```
Rule: skip-link
Impact: moderate
The skip-link target should exist and be focusable (https://dequeuniversity.com/rules/axe/3.5/skip-link?application=axeAPI)
```

You can find the issue on our repo [here](https://github.com/open-wc/open-wc/issues/1378). We think this may have been a regression in axe-core, and we'd be interested to hear your thoughts on this and how to move forward with this 😊

**Comment by straker** [comment]
Thanks for letting us know. I believe this is a similar problem from #2055

**Comment by straker** [comment]
Going to close this and track the work in #2055. A pr has been merged and we're about to create a release with the changes.
